Resources/doc/index.rst

```diff
@@ -568,279 +43,13 @@ and *Tree*.
 For more information on how to find and use these extensions, see the cookbook
 article about `using common Doctrine extensions`_.
 
-.. _cookbook-mongodb-field-types:
-
-Doctrine Field Types Reference
-------------------------------
-
-Doctrine comes with a large number of field types available. Each of these
-maps a PHP data type to a specific `MongoDB type`_. The following are just *some*
-of the types supported by Doctrine:
-
-* ``boolean``
-* ``date``
-* ``file``
-* ``float``
-* ``int``
-* ``string``
-* ``timestamp``
-
-For more information, see Doctrine's `Mapping Types documentation`_.
-
-.. index::
```

Comment:
No clue what this does, removed :)

Reply:
Pinging @javiereguiluz - maybe they know?

Reply:
This information is used to generate the "analytical index". Here you can see it: https://symfony.com/doc/current/genindex.html  We don't publicize it because we're thinking of removing it and future doc parsers may not support it. It's very useful to create indexes for paper books, where you can display the page where some concept is explained.

Reply:
Thanks for the quick explanation! Would you recommend keeping it or is it ok if we drop this?